File: movie-booking/src/main/java/com/movie/booking/model/Ticket.java
package com.movie.booking.model;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;


@Data
@Entity
@NoArgsConstructor
@AllArgsConstructor
public class Ticket {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private int transactionId;
    private String movieName;
    private String theaterName;
    private long bookedSeats;
    private String username;
}

File: movie-booking-ui/src/app/model/ticket.ts
import { Injectable } from "@angular/core";

@Injectable({
    providedIn: 'root'
})
export class Ticket {
    transactionId = 0;
    
    movieName = "";
    
    theaterName = "";
    
    bookedSeats = 0;
    
    username = "";
    

    public get TransactionId() {
        return this.transactionId;
    }
    public set TransactionId(value) {
        this.transactionId = value;
    }

    public get MovieName() {
        return this.movieName;
    }
    public set MovieName(value) {
        this.movieName = value;
    }

    public get TheaterName() {
        return this.theaterName;
    }
    public set TheaterName(value) {
        this.theaterName = value;
    }

    public get BookedSeats() {
        return this.bookedSeats;
    }
    public set BookedSeats(value) {
        this.bookedSeats = value;
    }

    public get Username() {
        return this.username;
    }
    public set Username(value) {
        this.username = value;
    }


}
